pod/cut/utils.py

```diff
@@ -0,0 +1,30 @@
+from pod.video.models import Notes, AdvancedNotes
+from pod.chapter.models import Chapter
+from pod.completion.models import Overlay, Track
+
+
+def clean_database(video_id):
```

Comment:
Ajouter un pydoc à chaque nouvelle fonction
de la forme """Describe what the function does."""